vaadin-combo-box-flow-parent/vaadin-combo-box-flow/src/main/java/com/vaadin/flow/component/combobox/ComboBox.java

```diff
@@ -714,9 +714,11 @@ public void setDataProvider(DataProvider<T, String> dataProvider) {
      * adjust its size until the backend runs out of items.
      * Usage example:
      * <p>
-     * {@code comboBox.setItems((filter, offset, limit) ->
-     *                      orderService.getOrders(filter, offset, limit),
-     *                                    textFilter -> new Order(textFilter));}
+     * {@code comboBox.setItemsWithFilterConverter(
+     *                 query -> orderService.getOrdersByCount(query.getFilter(),
+     *                                                        query.getOffset,
+     *                                                        query.getLimit()),
+     *                 orderCountStr -> Integer.parseInt(orderCountStr));}
```

Comment:
Add a comment that the orderCountStr validation is omitted.

Reply:
Done.